### ISSUE TITLE
How do I get the file "nalist.npy"?

### ISSUE THREAD

**Issue by snowy199** [AnasEmad11/C2FPL#3]
nalist = np.load("nalist.npy") #UCF
#nalist = np.load("nalist_XD.npy") #XD
These two files are not in the project, how do I get these two files?

**Comment by Daan-UT** [comment]
Did you ever figure this out?

**Comment by AnasEmad11** [comment]
Hello,

Sorry for the delay, Please find attached the nalist.npy for UCF-Crime.
________________________________

**Comment by Daan-UT** [comment]
I'm sorry but I see no attachment to your message @AnasEmad11

**Comment by snowy199** [comment]
> Did you ever figure this out?

Yes, I was recently reproducing the author's work, and I realized that this list refers to the start and end position of each video feature after concat, but this varies with the order in which the video features are concatenated

**Comment by Daan-UT** [comment]
How did you then find these boundaries in the concatenated list? I have another set of I3D features which are longer when concatenated than these ones, so I'd have to know the batch size

**Comment by snowy199** [comment]
> How did you then find these boundaries in the concatenated list? I have another set of I3D features which are longer when concatenated than these ones, so I'd have to know the batch size

I use [xd](https://github.com/Roc-Ng/XDVioDet/blob/master/list/rgb.list)'s list, which is the features of each video, just load them individually, then record their lengths and splice them together, for example the first feature is [97,5,1024],the second one is [100,5,1024], and after concat it's [197,5,1024]] then the first segment is [0,96] and the second one is [97,196]

**Comment by AnasEmad11** [comment]
> I'm sorry but I see no attachment to your message @AnasEmad11

I have added it to ./list in the repo

**Comment by JonathanFlores2503** [comment]
> > I'm sorry but I see no attachment to your message @AnasEmad11
> 
> I have added it to ./list in the repo

Hello, Thank you for sharing the `nanlist.npy` file. However, upon reviewing it, I noticed a discrepancy: the `concat_UCF_V2.npy` file provided contains only 779780 features [779780,10,2048], while the `nanlist.npy` file mentions 779951 features. Could you please provide the nanlist corresponding to the features of `concat_UCF_V2.npy` or advise me on how to use the shared `nanlist.npy` file? I appreciate your assistance.

**Comment by mohdwarish483** [comment]
> > How did you then find these boundaries in the concatenated list? I have another set of I3D features which are longer when concatenated than these ones, so I'd have to know the batch size
> 
> I use [xd](https://github.com/Roc-Ng/XDVioDet/blob/master/list/rgb.list)'s list, which is the features of each video, just load them individually, then record their lengths and splice them together, for example the first feature is [97,5,1024],the second one is [100,5,1024], and after concat it's [197,5,1024]] then the first segment is [0,96] and the second one is [97,196]



> > Did you ever figure this out?
> 
> Yes, I was recently reproducing the author's work, and I realized that this list refers to the start and end position of each video feature after concat, but this varies with the order in which the video features are concatenated

@snowy199 **could you please share the file to create nalist.npy  file**
